changelog_unreleased/handlebars/pr-7157.md

```diff
@@ -0,0 +1,23 @@
+#### handlebars: Improve MustacheStatement printing ([#7157](https://github.com/prettier/prettier/pull/7157) by [@dcyriller](https://github.com/dcyriller))
+
+<!-- prettier-ignore -->
+```hbs
+{{!-- Options --}}
+--print-width=40
+
+{{!-- input --}}
+<p>Hi  {{firstName}}  {{lastName}}   , welcome!</p>
+
+{{!-- stable --}}
+<p>
+  Hi {{firstName}} {{lastName
+  }} , welcome!
+</p>
+
+{{!-- master --}}
+<p>
+  Hi {{firstName}} {{
+    lastName
+  }} , welcome!
+</p>
```

Comment:
Can we add example without `width: 40`?

Reply:
Sure 👍

Reply:
I've updated the PR's description + the CHANGELOG to take a more regular example.